documentation/installation.md

```diff
@@ -5,7 +5,7 @@ For an in-depth guide see [the Xamarin docs for Calabash installation](https://d
 
 ### Calabash Sandbox
```

Comment:
Delete this entire section.

The Calabash Sandbox is not a supported project.

Reply:
Done